CMakeModules/wasm.cmake

```diff
@@ -121,7 +121,7 @@ macro(add_wast_target target SOURCE_FILES INCLUDE_FOLDERS DESTINATION_FOLDER)
 
   add_custom_command(OUTPUT ${DESTINATION_FOLDER}/${target}.wast
     DEPENDS ${target}.s
-    COMMAND ${BINARYEN_BIN}/s2wasm -o ${DESTINATION_FOLDER}/${target}.wast -s 1024 ${target}.s
+    COMMAND ${BINARYEN_BIN}/s2wasm -o ${DESTINATION_FOLDER}/${target}.wast -s 8192 ${target}.s
```

Comment:
We should consider making this something each contract can override.  Setting 1 true stack size for all contracts seems overly limiting.

Reply:
Yeah I agree, but I'd like @bytemaster 's input, since I am not sure how we want to handle this.  Any wasm *.cpp file that included memory.hpp would need to have it set, or else we need to set the memory space allocated based on the user stack.  So I'll keep this as an open issue for now (while I fix everything else), unless you have more input on this.